crates/re_renderer/src/renderer/rectangles.rs

```diff
@@ -65,6 +48,60 @@ pub enum TextureFilterMin {
     // TODO(andreas): Offer mipmapping here?
 }
 
+/// Describes a texture and how to map it to a color.
+pub struct ColormappedTexture {
+    pub texture: GpuTexture2DHandle,
+
+    /// Min/max range of the values in the texture.
+    /// Used to normalize the input values (squash them to the 0-1 range).
+    pub range: [f32; 2],
+
+    /// Raise the normalized values to this power (before any color mapping).
+    /// Acts like an inverse brightness.    /// Default: 1.0
+    pub gamma: f32,
+
+    /// For any one-component texture, you need to supply a color mapper,
+    /// which maps the normalized `.r` component to a color.
+    pub color_mapper: Option<ColorMapper>,
```

Comment:
should document here what happens when there's no color map for a single component texture and what happens when there is one for a multi-component one